deploy.py

```diff
@@ -866,6 +868,11 @@ def run(self) -> int:
               "configured yet.")
       return 0
 
+    if "vendor_hid" in self.args.features:
+      # vendor_hid as a work in progress and is not compatible with configure
```

Comment:
nit: as -> is
but feel free to ignore and we'll fix later

Reply:
ACK.